packages/channel-client/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function sleep(time: number) {
```

Comment:
What's the point of a fake delay?

Reply:
To fake a delay to simulate two users interacting?

Reply:
I think we were using this in RPS so that the user still sees all of the intermediate screens during interaction with the fake wallet.

Reply:
It's just faking the lag in messages being sent between participants? There is UI for this lag?